back/spec/models/basket_spec.rb

```diff
@@ -28,9 +28,10 @@
 
   context 'when a basket has more than the maximum votes' do
     before do
-      project = create(:continuous_budgeting_project, voting_max_total: 1000)
+      project = create(:continuous_budgeting_project)
```

Comment:
I wonder if it could make sense to pass the phase attributes to the factory, to avoid doing `project.phases.first.update!(...)` afterwards.

Reply:
Yeah, this was the first file I did before I added the phase attributes to the factory